go-controller/pkg/ovn/port_cache.go

```diff
@@ -90,3 +93,14 @@ func (c *portCache) remove(logicalPort string) {
 		}
 	}()
 }
+
+func (c *portCache) getValueOfNamedPort(namedPort string) (int32, error) {
```

Comment:
I think this needs to return a list of all the integer ports for a named port as there could be multiple